fdm-calculator/src/balance/organic-matter/types.d.ts

```diff
@@ -0,0 +1,330 @@
+ * Represents the organic matter supply from various fertilizer applications, categorized by type.
+ * All values are in kilograms of effective organic matter per hectare (kg EOM / ha).
+ * Effective Organic Matter (EOM) is the portion of organic matter that remains in the soil after one year.
+ *
+ * @see {@link https://www.agro-effluent.be/files/documents/2021-12-16-14-11-20-eindverslag-agro-effluent-17-06-2020.pdf} for more details on EOM.
+ */
+export type OrganicMatterSupplyFertilizers = {
+    /**
+     * The total amount of effective organic matter supplied by all types of fertilizers combined.
+     */
+    total: Decimal
+    /**
+     * The effective organic matter supply specifically from manure.
+     */
+    manure: {
+        /**
+         * The total amount of effective organic matter supplied by manure.
+         */
+        total: Decimal
+        /**
+         * A detailed list of individual manure applications.
+         * Each entry includes the application's unique identifier (`id`) and the amount of EOM supplied (`value`).
+         */
+        applications: { id: string; value: Decimal }[]
+    }
+    /**
+     * The effective organic matter supply specifically from compost.
+     */
+    compost: {
+        /**
+         * The total amount of effective organic matter supplied by all compost applications.
+         */
+        total: Decimal
+        /**
+         * A detailed list of individual compost applications.
+         * Each entry includes the application's unique identifier (`id`) and the amount of EOM supplied (`value`).
+         */
+        applications: { id: string; value: Decimal }[]
+    }
+    /**
+     * The effective organic matter supply from fertilizers other than manure or compost.
+     */
+    other: {
+        /**
+         * The total amount of effective organic matter supplied by all other fertilizer applications.
+         */
+        total: Decimal
+        /**
+         * A detailed list of individual other fertilizer applications.
+         * Each entry includes the application's unique identifier (`id`) and the amount of EOM supplied (`value`).
+         */
+        applications: { id: string; value: Decimal }[]
+    }
+}
+
+/**
+ * Represents the organic matter supply from cultivations (main crops and green manures).
+ * All values are in kilograms of effective organic matter per hectare (kg EOM / ha).
+ */
+export type OrganicMatterSupplyCultivations = {
+    /**
+     * The total amount of effective organic matter supplied by all cultivations on the field.
+     */
+    total: Decimal
+    /**
+     * A detailed list of cultivations that supply organic matter.
+     * Each entry includes the cultivation's unique identifier (`id`) and the amount of EOM supplied (`value`).
+     */
+    cultivations: { id: string; value: Decimal }[]
+}
+
+/**
+ * Represents the organic matter supply from crop residues.
+ * All values are in kilograms of effective organic matter per hectare (kg EOM / ha).
```

Comment:
the kluit en perspotjes will become a separate supply category besides fertilizer, crop residue?